### ISSUE TITLE
Member2 remains on the raft if new task is given while Member1 is getting diving gear

### ISSUE THREAD

**Issue by olandra** [olandra/CBSafe-Simulation#7]
If Member2 is waiting on the raft while Member1 is getting the diving gear and the team is given a new task, Member2 remains on the raft.

**Comment by olandra** [comment]
Fixed in Changeset 290
